documentation/modules/exploit/windows/fileformat/adobe_geticon.md

```diff
@@ -0,0 +1,67 @@
+## Vulnerable Application
+
+This module exploits a buffer overflow in Adobe Reader and Adobe Acrobat. Affected versions include < 7.1.1, < 8.1.3, and < 9.1. By creating a specially crafted pdf that a contains malformed Collab.getIcon() call, an attacker may be able to execute arbitrary code.
+
```

Comment:
can you add in the list of targets that were tested: https://github.com/rapid7/metasploit-framework/blob/master/modules/exploits/windows/fileformat/adobe_geticon.rb#L50